modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/ckeditor.jsp

```diff
@@ -624,10 +624,15 @@ name = HtmlUtil.escapeJS(name);
 
 		ckEditor.on('drop', function (event) {
 			var data = event.data.dataTransfer.getData('text/html');
-			var fragment = CKEDITOR.htmlParser.fragment.fromHtml(data);
-			var name = fragment.children[0].name;
-			if (name) {
-				return this.pasteFilter.check(name);
+
+			if (data) {
```

Comment:
Would we need to do this too for `<ClassicEditor>`? Maybe we want to move this elsewhere so it works there too for when we migrate?

Reply:
Yes, we still need to migrate the whole block to `ClassicEditor`, or change it completely. See [below](https://github.com/liferay-frontend/liferay-portal/pull/216#discussion_r466910674).

Reply:
Yeah, that's what I was thinking... I was wondering why something like this can't also live inside the plugin... 🤔

Reply:
Either that or let's copy-paste the thing in `ClassicEditor` already, so we don't get caught by surprise in the future :)